x-pack/plugins/apm/common/service_runtime_name.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function isJRubyServiceRuntimeName(serviceName?: string) {
+  return serviceName?.toLowerCase() === 'jruby';
+}
```

Comment:
I'm wondering if it would be better to include both `agent.name` and `runtime.name`:

```suggestion
export function isJRuby(agentName?: string, serviceRuntimeName?: string) {
  return agentName === 'ruby' && serviceRuntimeName === 'jruby';
}
```

@AlexanderWert any opinion on this?

Reply:
@AlexanderWert is on PTO, I'm going to add it, I don't think that's going to be a problem.